scripts/clean.sh

```diff
@@ -2,9 +2,11 @@
 
 echo "this is clean.sh"
 
+# Remove .pyc files
 find ./ -type f -name "*.pyc" -exec rm {} \;
-rm -rf ./google-breakpad/ ./build/ ./breakpad/ ./stackwalk breakpad.tar.gz
 
+# Remove any mdsw build artifacts
+rm -rf ./google-breakpad/ ./build/ ./breakpad/ ./stackwalk breakpad.tar.gz
```

Comment:
I don't know why github thinks this line changed--it didn't. I just added a couple of comments to this file.